### DISCUSSION TITLE
3.5.0 (20029) Release Candidate 1

### DISCUSSION THREAD

**Discussion by Jason-Morcos** [Sequel-Ace/Sequel-Ace#1383] [General]
## App Store Release Notes
- Tables can now be pinned to the top of the table list by right clicking on the name of the table
- The structure view is now searchable and sortable
- Changed default query editor font to Menlo to improve query editor performance
- Improved tab coloring appearance in Dark and Light modes
- When exporting, clicking S[tructure] C[ontent] or D[rop] in the export window will toggle the value for all tables
- Views no longer have a funky fake table placeholder generated when exporting as SQL
- SSL connection status is now more accurately displayed in the tab bar (a lock indicates you are connected via SSL)
- Disabled TouchBar font panel API to improve query editor performance
- Fixed a crash with editing columns with non-string (ex. Geometry Point) data

### Added
- 49ed5429, Support for pinning tables at top of tables list 
- 18af48c0,  Clicking S[tructure] C[ontent] or D[rop] in export window will toggle the value for all tables
- 7293fc50,   Filter and Sorting to Structure "fields" Table View.

### Fixed
- 2f33050, Fixed a crash with editing columns with non-string data
- 7e16cee2, Disable dark mode variants of colors for tab bar 
- d46f0712, Disable TouchBar font panel API 

### Changed
- 8afb7e51,  Improved tab coloring appearance in Dark and Light modes
- 81ec264f,  Stop exporting a placeholder table for views in database exports
- da43e08c,  Accurately show SSL connection status always
- a37b653c,  Allow sorting every column on the Structure "fields" table view
- b002cae1,  Change default query editor font to Menlo to improve query editor performance

### Infra

<hr /><em>This discussion was created from the release <a href='https://github.com/Sequel-Ace/Sequel-Ace/releases/tag/production/3.5.0-20029'>3.5.0 (20029) Release Candidate 1</a>.</em>

**Comment by samuelbradshaw** [comment]
This is shaping up to be a great release!

**Comment by d42ohpaz** [comment]
Thank you for all of your hard work!

**Comment by madebycaliper** [comment]
Love to see the searchable/sortable structure view!! Thanks for keeping up with this project
